src/plugins/hosts/DrawerPanel/styles.js

```diff
@@ -15,10 +15,7 @@ export const drawerPanelStyles = (isLeft, isOpen) =>
       [isLeft ? "marginRight" : "marginLeft"]: "auto",
       width: isOpen ? 340 : "auto",
       height: "100%",
-      "& .MuiBackdrop-root": {
-        display: "none"
-      },
-      "& .MuiDrawer-paper": {
+      "& > *": {
```

Comment:
what happened here?
What we had before was more specific, targeted all `.MuiBackdrop-root` and `.MuiDrawer-paper` that were descendants of `drawer`.
Your change targets all direct descendants of `drawer`. 
So it's different. Are we sure it won't affect anything in the future? Or now, for that matter.

Reply:
That wasn't applying the styles properly. That is why it was changed.

Remember when you showed me there was a problem in CE?

that was the problem